source/_posts/2023-09-06-release-20239.markdown

```diff
@@ -134,41 +120,98 @@ Other more advanced fields, like attributes, are not available from the UI. You
+We added a new service for weather entities, `weather.get_forecast`, that responds with the weather forecast.
 
 Previously you would have to look at the `forecast` attribute of a weather entity to get the forecast, now this can be done by calling a new service, `weather.get_forecast`, that will respond with the forecast.
-You can also specify what type of forecast you want, daily or do you want to know it for every hour? This means you no longer need separate entities for every forecast type, but it can all be done in 1 entity.
-
-If an entity supports multiple forecasts, you can choose which one you want to see in the more info dialog.
-In the weather card in your dashboard, you can also now pick what kind of weather forecast you want to show in that card.
+You can also specify what type of forecast you want, do you want to see the forecast by day or do you want to know it for every hour? This means you no longer need separate entities for every forecast type, but it can all be done in 1 entity.
 
 All weather integrations are updated to support this new service.
 
-With this change, the `forecast` attribute of the weather entity is deprecated, and will be removed in Home Assistant Core 2024.3.
+<p class='img'>
+<img src='/images/blog/2023-09/get-forecast-service.png'></a>
+Temporary screenshot that needs to be replaced.
+</p>
 
-You can use the newly added `action` part of a template entity to fetch the forecast and use it in your template.
+If an entity supports multiple forecasts, you can choose which forecast you want to see in the more info dialog.
+In the weather card in your dashboard, you can also now pick what kind of weather forecast you want to show in that card.
 
 <p class='img'>
-<img src='https://media.discordapp.net/attachments/427516175237382144/1146445861421846608/SPOILER_CleanShot_2023-08-30_at_16.03.542x.png?width=1466&height=794'></a>
+<img src='weather-more-info.png'></a>
 Temporary screenshot that needs to be replaced.
 </p>
 
+With this change, the `forecast` attribute of the weather entity is deprecated, and will be removed in Home Assistant Core 2024.3.
+This will make it possible to have on-demand forecasts, that update when you want them to, instead of them being updated all time. It will also make the system faster. 
+
+You can use the newly added `action` part of a template entity to fetch the forecast and use it in your template:
+
+{% raw %}
+
+```yaml
+template:
+  - trigger:
+      - platform: time
+        at: "00:00:00"
+    action:
+      - service: weather.get_forecast
+        target:
+          entity_id: weather.home
+        data:
+          type: daily
+        response_variable: forecast
+    binary_sensor:
+      - name: Is it going to rain?
+        state: "{{ forecast.forecast[0].precipitation > 0 }}"
+```
+
+{% endraw %}
+
 ## Onboarding
 
 Our onboarding got a fresh new look and it is now easier to get started with Home Assistant!
```

Comment:
```suggestion
While it is certain that Home Assistant is a great tool once you understand how it works, one part that was often overlooked is its associated learning curve.

We are now actively working to flatten that curve so that you can get more from your home faster and simpler. 
This release shows the first result of that effort, a new onboarding flow.
```